styles/language/Dutch/language.php

```diff
@@ -688,7 +688,7 @@
     // Other
     'page_loaded_in' => 'Pagina geladen in {x} seconden', // Don't replace {x}; 's' stands for 'seconds'
     'none' => 'Geen',
-    '404' => 'Sorry, we konden deze pagina niet vinden. Weet je zeker dat je de URL correct heeft gespeld?'
+    '404' => 'Sorry, we kunnen de pagina niet vinden die je probeert te bezoeken. Controleer de URL en probeer het opnieuw, als het nog niet lukt neem dan contact op met de Administrator'
```

Comment:
Just want to make sure if this translation is correct. Any updates on that? @TheSander562 @rodymol123